### PR TITLE
Disable delete collections for KUZZLE V1

### PR DESCRIPTION
**Pull request by morgandruesne**
## What does this PR do ?
Disable the possibility in the UI to delete collections with an environment for kuzzle v1

### CODE REVIEW

File: src/components/Data/Collections/CollectionList.vue [hunk 1]
```diff
@@ -59,7 +60,12 @@
                 <b-button
                   variant="outline-danger"
                   :disabled="!bulkDeleteEnabled"
-                  @click="DeleteCollections"
+                  v-if="
+                    $store.state.kuzzle.environments[
+                      $store.state.kuzzle.currentId
+                    ].backendMajorVersion !== 1
+                  "
+                  @click="deleteCollections"
```
Comment (xbill82) [inline src/components/Data/Collections/CollectionList.vue:68]:
```suggestion
                  v-if="
                    $store.getters.kuzzle.currentEnvironment.backendMajorVersion !== 1
                  "
                  @click="deleteCollections"
```

File: src/components/Data/Collections/CollectionList.vue [hunk 2]
```diff
@@ -190,6 +196,10 @@
             <b-button
               class="mx-1"
               variant="link"
+              v-if="
+                $store.state.kuzzle.environments[$store.state.kuzzle.currentId]
```
Comment (xbill82) [inline src/components/Data/Collections/CollectionList.vue:200]:
```suggestion
                $store.getters.kuzzle.currentEnvironment
```

File: src/components/Data/Collections/CollectionList.vue [hunk 3]
```diff
@@ -230,6 +240,10 @@
           >Cancel</b-button
         >
         <b-button
+          v-if="
+            $store.state.kuzzle.environments[$store.state.kuzzle.currentId]
```
Comment (xbill82) [inline src/components/Data/Collections/CollectionList.vue:244]:
```suggestion
            $store.getters.kuzzle.currentEnvironment
```

**Review by xbill82 — CHANGES_REQUESTED** [review]
Getter is better 😎

**Review by xbill82 — APPROVED** [review]

**Review by xbill82 — CHANGES_REQUESTED** [review]
Could you add an e2e test, please?

**Review by berthieresteban — APPROVED** [review]

**Review by Njuelle — APPROVED** [review]

**Review by xbill82 — APPROVED** [review]

**Pull request merged**